jobs/scripts/nightly-builds/nightly-builds.sh

```diff
@@ -11,8 +11,13 @@ set -e
 
 BUILDREQUIRES="libaio-devel libattr-devel libxml2-devel readline-devel openssl-devel fuse-devel glib2-devel userspace-rcu-devel libacl-devel libuuid-devel gperftools-devel gperftools-libs"
 
-if [ "${CENTOS_VERSION}" -eq 8 ]
-then
+if [ "${CENTOS_VERSION}" == "8-stream" ]; then
+    ENABLE_REPOS="--enablerepo=powertools"
+    BUILDREQUIRES="${BUILDREQUIRES} python3-devel rpcgen libtirpc-devel liburing-devel rsync "
+    yum -y update
```

Comment:
this one makes me slightly worried, as it may mean the build server is a moving target, with Stream. It's not bad, in the sense that we might catch issues coming from Stream and impact us faster, but we could do it with the container builds as well. And it takes time. If it's not critically needed, we can remove it, I think.

Reply:
Thanks for the input Yaniv. The logic was introduced due to the package differences between centos-7 and 8. Now that 8 is EOL we might have to keep the logic for 8-stream is what was I thought.
I will try another logic and see if I can do it differently.

Reply:
From what I understand if we make changes to the **Containerfile** the package dependency would be still there in the **Duffy** nodes that we procure. So the above logic will install all the packages in the node we get for rpm build. I will check with others to see if I have understood correctly and if it can be done via the container builds I will do it.

Reply:
I have tested these changes on a test job https://ci.centos.org/view/Gluster/job/test-gluster_build_rpms/, by pulling the changes from this PR and it works as expected.